src/modules/SystemSettings.rb

```diff
@@ -128,8 +130,13 @@ def Activate
         # set the scheduler
         Bootloader.modify_kernel_params("elevator" => new_elevator)
 
-        # TODO FIXME: set the scheduler for all disk devices,
-        # reboot is required to activate the new scheduler now
+        # activate the scheduler for all disk devices
+        if new_elevator != :missing
```

Comment:
so if new scheduler is missing then keep old value? I am just making sure that you do not want to write default